index.js

```diff
@@ -241,10 +201,7 @@ function resolve_array(arr) {
  * @return {String}
  */
 
-function type(v) {
-  return isArray(v)
-    ? 'array'
-    : v.vo
-    ? 'vo'
-    : typeof v;
+function typeOf(v) {
```

Comment:
hm, u have some right. maybe limits to not come instance of class. e.g. `vo(new Fn)` which, yea, im not sure should we handle it or not?